schemas/json/layout/layoutSettings.schema.v1.json

```diff
@@ -77,6 +80,11 @@
       "additionalProperties": false,
       "description": "An attribute specifying if the translation dropdown menu should be visible",
       "type": "boolean"
+    },
+    "pdfLayoutName": {
+      "additionalProperties": false,
+      "description": "Name of a custom layout file to use for PDF creation instead of the automatically generated PDF.",
+      "type": "string"
     }
```

Comment:
How would page-ordering work with different modes? PDF is completely separate from the rest of the pages so it would need its own pageordering to support multiple layout files then?

Reply:
But to your first point, yes, it should simply be omitted from page-order and the layout will never be accessible through regular navigation

Reply:
That's a very good point about internal page-ordering when having multiple PDF pages.. There's actually a (undocumented) alternative way of ordering pages, where you can specify (in the layout file itself) what is the next/prev page to go to from that point on. If that was documented, and could be controlled using expressions, it would be much more powerful than the current tracks functionality. (Look for the `INavigationConfig` type if want to unravel that thread.)